nixos/modules/services/networking/bird.nix

```diff
@@ -4,6 +4,7 @@ let
   inherit (lib) mkEnableOption mkIf mkOption optionalString types;
 
   cfg = config.services.bird2;
+  caps = [ "CAP_NET_ADMIN" "CAP_NET_BIND_SERVICE" "CAP_NET_RAW" ];
```

Comment:
According to https://bird.network.cz/?get_doc&v=20&f=bird-1.html#ss1.4, CAP_NET_* is enough for bird's operation, and CAP_NET_BROADCAST is unused as documented in https://man7.org/linux/man-pages/man7/capabilities.7.html. This also brings us in line with unit file from Arch Linux.